homeassistant/components/eq3btsmart/entity.py

```diff
@@ -0,0 +1,18 @@
+"""Base class for all eQ-3 entities."""
+
+from eq3btsmart.thermostat import Thermostat
+
+from homeassistant.helpers.entity import Entity as HassEntity
```

Comment:
```suggestion
from homeassistant.helpers.entity import Entity
```

Importing it as a different name makes this much harder for maintainers for find these